actions/delete_node_handler.go

```diff
@@ -156,10 +159,44 @@ func (h *deleteNodeHandler) Handle(ctx context.Context, action *castai.ClusterAc
 			if len(pods.Items) > 0 {
 				return true, fmt.Errorf("waiting for %d pods to be terminated on node %v", len(pods.Items), req.NodeName)
 			}
+			// Check if there are any volume attachments left for the node, but don't block on waiting for them.
+			volumeAttachments, err := h.clientset.StorageV1().VolumeAttachments().List(ctx, metav1.ListOptions{})
```

Comment:
Does this filter by node? The filter is different from below and there we manually select the node. Wouldn't it get all volume attachments?

Reply:
yes, it would, unfortunately there are no node filters for node attachments.